src/csvcubed/utils/validations.py

```diff
@@ -0,0 +1,90 @@
+# This script contain a set of function that can be used to validate specific class attributes/ member variables
+from typing import Optional
+from csvcubed.models.validationerror import ValidateModelProperiesError
+
+from typing import List, TypeVar, Type, Callable
+
+from csvcubed.utils.uri import looks_like_uri
+
+
+T = TypeVar("T")
+
+
+def validate_list(
+    validate_list_item: Callable[[T], List[ValidateModelProperiesError]],
+) -> Callable[[List[T], str], List[ValidateModelProperiesError]]:
```

Comment:
Can you add docstring comments to all of these functions to explain what they do?

Reply:
done